presto-hive/src/test/java/io/prestosql/plugin/hive/benchmark/FileFormat.java

```diff
@@ -293,52 +419,65 @@ public boolean supports(TestData testData)
         return true;
     }
 
-    private static ConnectorPageSource createPageSource(
+    private static ConnectorPageSource createPageSourceForBaseColumns(
```

Comment:
I had added `ForBaseColumns` to indicate that the provided column names to this method will always correspond to top-level (or base) columns, not projected ones.

For example, `createFileFormatReader` should not be given a projected column (like "a.b"), since the `createPageSource`  method is only capable of reading "a", and will leave up to the hive page source wrapper to extract out the field.  `createGenericReader` uses the column handles constructed for such nested fields.